lib/sles4sap.pm

```diff
@@ -32,6 +32,7 @@ use Carp qw(croak);
 our @EXPORT = qw(
   $instance_password
   $systemd_cgls_cmd
+  $resource_aliase
```

Comment:
It should be either `$resource_alias` or `$resource_aliases`

Reply:
Fixed. Thanks